test/test_transforms.py

```diff
@@ -681,15 +681,224 @@ def test_convert_image_dtype_int_to_int_consistency(self):
                     self.assertEqual(actual_min, desired_min)
                     self.assertEqual(actual_max, desired_max)
 
-    @unittest.skipIf(accimage is None, 'accimage not available')
+
+    def test_affine(self):
```

Comment:
We should try to move these methods out of the class  and just have pure functions at the bottom of the file. In the code below there are a lot of references to `self` which we want to remove and replace as per the instrucitons in #3945  :)